functions/test/utils/IPChecker.test.ts

```diff
@@ -0,0 +1,63 @@
+import {secretManager} from '../../src/services';
+import IPChecker from '../../src/utils/IPChecker';
+import { expect } from 'chai';
+import {anything, spy, when} from 'ts-mockito';
+import {Netmask} from 'netmask';
+
+function mockSecretManager(netmasks: String[]) {
+    const spiedSecretManager = spy(secretManager);
+    when(spiedSecretManager.getConfig(anything())).thenResolve(netmasks);
+}
+
+describe('IPChecker init', () => {
```

Comment:
place one describe at the top, and one describe for each method being tested - to split up what is being tested